.circleci/config.yml

```diff
@@ -598,7 +598,7 @@ jobs:
             - front-libs-{{ .Environment.CACHE_VERSION }}-{{ checksum "packages-last-commit.txt" }}
       - run:
           name: Run test on standalone site
-          command: yarn workspace standalone_site run test
+          command: yarn workspace standalone_site run test:CI
```

Comment:
```suggestion
          command: yarn workspace standalone_site run test --watchAll=false --maxWorkers=2
```

Reply:
The rebase de "feature/anthony/create-classroom" will this.